source/release-notes/v1.6-release-notes.rst

```diff
@@ -0,0 +1,178 @@
+
+  .. highlight:: none
+
+  ::
+
+    - Discovered several implementation issues (bugs) that could affect the
+      proper operation of Open OnDemand. 
+    - No major issues were found in a comprehensive evaluation of the
+      architecture and critical resources in Open OnDemand, and analysis of
+      potential code weaknesses in critical components.
+    - Found a potential vulnerability, dormant based on current configuration
+      and launch settings.
+
+The potential security issue was the ability for a user to consume all available resources on a machine using a `fork bomb`_ or similar attack. OnDemand is designed to allow users to do anything that they are permitted to do on a system; to not 'hide the machine' as our PI likes to say. The OnDemand team expects that HPC centers are using batch schedulers which are configured to limit user resource utilization with tools like ``ulimit`` or ``cgroups``. After a conversation about those assumptions TrustedCI agreed that the ability of a user to run a fork bomb was not an issue with OnDemand.
+
+ To address the bugs that were found three issues were opened on Github.
+
+  - Creating a new directory whose name includes HTML/XML tags results in unintended behavior (`ood-fileexplorer#198`_)
+  - Datatables does not handle long job names well (`ood-myjobs#290`_)
+  - Default "simple sequential job" is reported to have not execute permissions on job script (`ood-myjobs#289`_)
+
+.. _ood-fileexplorer#198: https://github.com/OSC/ood-fileexplorer/issues/198
+.. _ood-myjobs#289: https://github.com/OSC/ood-myjobs/issues/289
+.. _ood-myjobs#290: https://github.com/OSC/ood-myjobs/issues/290
+
+The issue with the File Explorer is not a security concern has not been reported as a problem that real users have, and given that the OnDemand team intends to replace the File Explorer as part of the OnDemand 2.0 project the issue is one that will not be fixed for now. The File Explorer issue will be addressed in the next major version of that application. The other two issues-both relating to the Job Composer-have been addressed for the 1.6.x release.
+
+Thanks to Elisa Heymann, Joel Atkins and Barton P. Miller for performing the audit!
+
+.. _fork bomb: https://en.wikipedia.org/wiki/Fork_bomb
+.. _TrustedCI: https://trustedci.org/
+
+Added rendering of HTML or Markdown in Job Composer template manifests
+......................................................................
+
+If the environment variable ``RENDER_TEMPLATE_NOTES_AS_MARKDOWN`` is set to a truthy value then job template manifests displayed in the Job Composer that are written in Markdown will be run through the RedCarpet renderer. HTML values will always be rendered as HTML. This will allow sites to embed images or links to documentation in their templates. (`ood-mjobs#291`_)
+
+.. _ood-mjobs#291: https://github.com/OSC/ood-myjobs/issues/278
+
+
+Added L18n hooks to the Dashboard and Job Composer
```

Comment:
[it is "I18n" and "L10n"](https://blog.mozilla.org/l10n/2011/12/14/i18n-vs-l10n-whats-the-diff/) not L18n.

But this is also technically incorrect.  "Added localization options to the Dashboard and Job Composer" might be better. The problem with the nomenclature is there is no single noun that represents the "feature we added". Rather, two verbs, one for adding the feature (internationalizing the app by abstracting locale-specific strings) and two for using the feature (localizing the app to your site by providing the translations).